cdap-common/src/main/resources/cdap-default.xml

```diff
@@ -1710,14 +1710,6 @@
   </property>
 
   <property>
```

Comment:
you will also have to update https://github.com/caskdata/cdap/blob/release/3.5/cdap-docs/admin-manual/build.sh#L22 with the MD5 of the file after this change